powerscore-entryboard/src/App.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
+import NotFound from './NotFound.vue';
+import Entryboard from './components/Entryboard.vue';
+import { ref, computed} from 'vue';
+
+const routes = {
+  '/entryboard': Entryboard,
```

Comment:
Is it possible that we can make the default route `/` point to the entry board instead? The goal (heh pun) of this UI is just to create the locker room assignments. Nothing else. And since it's only going to be showing those locker room assignments, it doesn't feel like there needs to be additional routes.